src/mongocxx/test/spec/crud.cpp

```diff
@@ -30,6 +30,19 @@ using namespace mongocxx::spec;
 using bsoncxx::builder::basic::kvp;
 using bsoncxx::builder::basic::make_document;
 
+bool is_unsupported(bsoncxx::stdx::string_view description) {
+    std::set<std::string> unsupported_tests = {"Deprecated count with empty collection",
+                                               "Deprecated count with collation",
+                                               "Deprecated count without a filter",
+                                               "Deprecated count with a filter",
+                                               "Deprecated count with skip and limit"};
```

Comment:
CXX-1931 removed the deprecated count helper (in favor of the non-deprecated countDocuments and estimatedDocumentCount). We cannot run these tests.

Reply:
Acknowledged.